clang/include/clang/Sema/SemaSYCL.h

```diff
@@ -14,9 +14,11 @@
 #define LLVM_CLANG_SEMA_SEMASYCL_H
 
 #include "clang/AST/Decl.h"
+#include "clang/AST/DeclBase.h"
```

Comment:
Wonder same about declbase as well, probably a bit of work on a few of these includes to do that sort of analysis.

Reply:
This one should be fixed as well now.

Reply:
> This one should be fixed as well now.

Can we drop `#include "clang/AST/Decl.h"` as well?

Reply:
I'm okay with that.
It just looks strange to add forward declaration for `Decl` class when full declaration is included with `#include "clang/AST/Decl.h"`. Probably the cleaner solution would be to drop forward declaration of `Decl` and add it in the follow-up patch.
Anyway, this is a minor comment and should not block the merge.